### ISSUE TITLE
Website (smilesdrawer.rocks) that is linked in Readme is offline

### ISSUE THREAD

**Issue by OBrink** [reymond-group/smilesDrawer#144]
"This site can’t be reached
www.smilesdrawer.rocks refused to connect."

**Comment by KrofDrakula** [comment]
The correct address is [smilesdrawer.rocks](http://smilesdrawer.rocks).

**Comment by OBrink** [comment]
Ok, thanks. The link in the readme ([https://www.smilesdrawer.rocks/](https://www.smilesdrawer.rocks/)) should be updated though

**Comment by KrofDrakula** [comment]
Yeah, just haven't had time to open a PR since that's not the only problem with the README stuff. 😅
